vividus-plugin-ssh/src/test/java/org/vividus/ssh/sftp/SftpCommandTests.java

```diff
@@ -168,14 +170,27 @@ void shouldExecuteGetCommand() throws IOException, SftpException
     @Test
     void shouldExecutePutCommand() throws IOException, SftpException
     {
-        ChannelSftp channel = mock(ChannelSftp.class);
         String content = "content";
-        String destination = "/remote/path";
+        InputStream inputStream = testPutCommand(SftpCommand.PUT, content);
+        assertEquals(content, new String(inputStream.readAllBytes(), StandardCharsets.UTF_8));
+    }
+
+    @Test
+    void shouldExecutePutFileCommand() throws IOException, SftpException
+    {
+        String filePath = ResourceUtils.loadFile(SftpCommand.class, "test.txt").getPath();
+        testPutCommand(SftpCommand.PUT_FROM_FILE, filePath);
```

Comment:
missing content verification (similar to the validation at line 175)

Reply:
Files.newInputStream(Paths.get(filePath))
Closes input stream too early - can't read argument-captured bytes on verification stage. Line 175 won't work. How else?

Reply:
use `argThat` instead of `ArgumentCaptor`, read the `inputStream` and verify content in the mockito matcher